lib/irb.rb

```diff
@@ -986,67 +866,6 @@
 #   If +line_no+ is a negative, the return value +line_no+ many lines before
 #   the most recent return value.
 #
-# ==== Example using IRB Sessions
```

Comment:
The example here has a heavy focus on the multi-irb feature we plan to deprecate, which is a bit misleading.